### DISCUSSION TITLE
migrating globaleaks to new debian

### DISCUSSION THREAD

**Discussion by FFD8FFE1** [globaleaks/globaleaks-whistleblowing-software#4275] [General]
hi i have created a new debian VM and i have followed the migration guide
https://docs.globaleaks.org/en/main/user/admin/UpgradeGuide.html

i have extracted the backup of the globaleaks folder to /var/globaleaks  the backup is from a globaleaks system ver. 5.0.17, and the database i detected as version 68

when i run the globalkeaks install script 5.0.18 it creates a lot of errors

 have tested that the install method works when there is no backup of old data.
wget https://deb.globaleaks.org/install-globaleaks.sh
chmod +x install-globaleaks.sh
./install-globaleaks.sh

i have attached the error log with masked url´s

i hope you can help me migrate, since the described method doesn't work in my case.

Thanks in advance
[migrate_error_log.txt](https://github.com/user-attachments/files/17458786/migrate_error_log.txt)

**Comment by evilaliv3** [comment]
Closing as duplicated for https://github.com/globaleaks/globaleaks-whistleblowing-software/issues/4276
